apollo-normalized-cache/src/jvmMain/kotlin/com/apollographql/apollo/cache/normalized/ApolloStore.kt

```diff
@@ -138,22 +137,20 @@ interface ApolloStore {
   </V></T></D> */
   fun <D : Operation.Data, V : Operation.Variables> read(
       operation: Operation<D, V>,
-      responseFieldMapper: ResponseFieldMapper<D>,
       responseNormalizer: ResponseNormalizer<Record>,
       cacheHeaders: CacheHeaders
   ): ApolloStoreOperation<Response<D>>
 
   /**
-   * Read GraphQL fragment from store.
+   * Read from store.
    *
-   * @param fieldMapper [ResponseFieldMapper] to be used for field mapping
    * @param cacheKey    [CacheKey] to be used to find cache record for the fragment
    * @param variables   [Operation.Variables] required for fragment arguments resolving
    * @param <F>         type of fragment to be read
    * @return {@ApolloStoreOperation} to be performed, that will be resolved with cached fragment data
   </F> */
-  fun <F : GraphqlFragment> read(
-      fieldMapper: ResponseFieldMapper<F>,
+  fun <F> read(
```

Comment:
we should probably leave only one API (remove `fun <D : Operation.Data, V : Operation.Variables> read(...)`

Reply:
Follow up PR for store APIs: https://github.com/apollographql/apollo-android/pull/2839